src/shared/ui/Items/SelectBase/SelectBaseText.tsx

```diff
@@ -15,10 +15,10 @@ export const SelectBaseText = (props: Props) => {
       lineHeight="28px"
       testid="select-text"
       sx={{
+        wordBreak: 'break-word',
         cursor: 'pointer',
-        lineBreak: 'anywhere',
+        hyphens: 'auto',
```

Comment:
This will still allow the browser is automatically break words at appropriate hyphenation points, following whatever rules it chooses. If the intent is to avoid breaking mid-word altogether, then we should use `[manual](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens#manual)` or `[none](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens#none)` instead.